src/ext/conversion/__test__/export-properties.spec.ts

```diff
@@ -83,6 +83,8 @@ describe('exportProperties', () => {
         qHyperCubeDef: {
           qDimensions,
           qMeasures,
+          qColumnOrder: [0, 2, 1],
+          columnOrder: [0, 1],
```

Comment:
we make qColumnOrder and columnOrder same here to migrate confusion?

Reply:
sure! I will fix that.